packages/api-kintone/src/invoice/getInvoiceByEstDataId.ts

```diff
@@ -0,0 +1,13 @@
+import { getRecords } from '../common';
+import { appId, RecordType } from './config';
+
+export const getInvoiceByEstDataId = async (estDataId: string) => {
+  if (!estDataId) throw new Error('Invalid estData id.');
+
+  const dataIdKey = 'dataId';
```

Comment:
```suggestion
    const dataIdKey : keyof RecordType['estimateLists']['value'][number]['value'] = 'dataId';
```

長くなりますが、DB側の変更がある際、より安全になります。

Reply:
ご指摘ありがとうございます。
[9bdc0db](https://github.com/Lorenzras/yumecoco-monorepo/pull/173/commits/9bdc0dbff072f452d64065224494b980d8c8a52b)にて修正しました。